packages/amplify-category-api/resources/awscloudformation/overrides-resource/APIGW/package.json

```diff
@@ -8,11 +8,9 @@
     "test": "echo \"Error: no test specified\" && exit 1"
   },
   "dependencies": {
-    "fs-extra": "^9.1.0"
+    "@aws-amplify/cli-extensibility-helper": "2.3.0-apiext3.0"
```

Comment:
I think this can still be in devDependencies

Reply:
Is this the latest tag?

Reply:
the we cant import types during runtime right ?

Reply:
following samme approach in ther categories